src/pages/Login.js

```diff
@@ -1,9 +1,45 @@
-import React from 'react'
+import { useRef } from "react"
+import {useNavigate} from "react-router-dom"
 
-const Login = () => {
-  return (
-    <div>Login</div>
+
+
+const Login = ({ login }) => {
+  const formRef = useRef()
+  const navigate = useNavigate()
+  const handleSubmit = (e) => {
+    e.preventDefault()
+      const formData = new FormData(formRef.current)
+      const data = Object.fromEntries(formData)
+      const userInfo = {
+        "user":{ email: data.email, password: data.password }
+      }
+      login(userInfo)
+      navigate("/")
+      e.target.reset()
+  }
+  return(
+    <div className="auth-body">
+          <h1 className="text-center my-5" style={{ textShadow: "0 5px 5px grey" }}>
+        Login
+      </h1>
+      <form className="w-50 m-auto p-3 shadow-lg rounded my-5 pt-5 pb-3" ref={formRef} onSubmit={handleSubmit}>
+        Email: <input className='form-control' type="email" name='email' placeholder="email" />
+        <br/>
+        Password: <input className='form-control' type="password" name='password' placeholder="password" />
+        <br/>
+        <select className="form-control text-center">
+          <option value="">-Choose Skill Level-</option>
+          <option value="1">1</option>
+          <option value="2">2</option>
+          <option value="3">3</option>        
+        </select>
```

Comment:
Do you need to choose a skill level to login, or is it just for signup?

Reply:
we are keeping them in both so they can change their skill level whenever signing in but we updated userinfo being passed.